x-pack/plugins/apm/public/plugin.ts

```diff
@@ -372,33 +370,26 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
           id: 'services',
           title: servicesTitle,
           path: '/services',
-          navLinkStatus: config.serverless.enabled
-            ? AppNavLinkStatus.visible
-            : AppNavLinkStatus.default,
```

Comment:
Does it mean that for serveless and default envs the value will be?
`visibleIn: ['globalSearch', 'sideNav']`

Reply:
Yes. Although in serverless the `'sideNav'` value is not used as it is [the navigation tree](https://github.com/elastic/kibana/blob/main/x-pack/plugins/serverless_observability/public/navigation_tree.ts) that defines the visibility of links in the side nav.

So to be correct, the default in serverless is `visibleIn: ['globalSearch']`